server/processors/src/main/java/org/hawkular/apm/server/processor/nodedetails/NodeDetailsDeriver.java

```diff
@@ -116,7 +123,8 @@ protected void deriveNodeDetails(Trace trace, List<Node> nodes, List<NodeDetails
                     nd.setHostName(trace.getHostName());
                 }
 
-                nd.setProperties(trace.allProperties());
```

Comment:
I thought there is a bug because [1]`Trace#getAllProperties()`  -> `Node#includeProperties()` (without descendants) [2], but `ContainerNode` [3] overrides it iterate over all its descendants.



[1]: https://github.com/hawkular/hawkular-apm/blob/master/api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java#L200
[2]: https://github.com/hawkular/hawkular-apm/blob/master/api/src/main/java/org/hawkular/apm/api/model/trace/Node.java#L233
[3]: https://github.com/hawkular/hawkular-apm/blob/master/api/src/main/java/org/hawkular/apm/api/model/trace/ContainerNode.java#L86